web/themes/custom/usagov/usagov.theme

```diff
@@ -557,3 +561,24 @@ function usagov_preprocess_breadcrumb(&$variables): void {
     $variables['breadcrumb'][] = ['text' => $currentTitle];
   }
 }
+
+function usagov_preprocess_header(array &$variables) {
+  $affiliate = \Drupal::state()->get('usagov_affiliate', 'usagov_en_all_gov');
+  $affiliate_es = \Drupal::state()->get('usagov_affiliate_es', 'usagov_es_internal');
+
+  $current_path = \Drupal::service('path.current')->getPath();
+  $variables['usagov_affiliate_lang'] = str_starts_with($current_path, '/es') ? $affiliate_es : $affiliate;
```

Comment:
Do we not have the language setting of the page available here? I think that if this were working, it would put the Spanish search on `/estimate-college-cost`.